pkg/api/controller.go

```diff
@@ -712,7 +712,7 @@ func (c *Controller) CreatePolicy(w http.ResponseWriter, r *http.Request, body C
 		Statement:   stmts,
 	}
 
-	err := c.Auth.WritePolicy(ctx, p)
+	err := c.Auth.WritePolicy(ctx, p, false)
```

Comment:
I assume we will need to update the UI as this is the partial fix that we implemented before.